config.yml.erb

```diff
@@ -542,3 +542,9 @@ dashboard_reporting_db_reader: '<%="#{reporting_db_reader}#{dashboard_db_name}"%
 dashboard_reporting_db_writer: '<%="#{reporting_db_writer}#{dashboard_db_name}"%>'
 pegasus_reporting_db_reader:   '<%="#{reporting_db_reader}#{pegasus_db_name}"%>'
 pegasus_reporting_db_writer:   '<%="#{reporting_db_writer}#{pegasus_db_name}"%>'
+
+# Configurations for making calls to the Immersive Reader API on Azure
+imm_reader_tenant_id:
+imm_reader_client_id:
+imm_reader_client_secret: !Secret
+imm_reader_subdomain:
```

Comment:
If these constant values will continue to be the same across environments, I'd suggest specifying them once here in the base config and nothing else in the environment-specific configuration files, to reduce duplication:

```suggestion
# Configurations for making calls to the Immersive Reader API on Azure
imm_reader_tenant_id: '01521c57-6696-4ac9-b8ba-d2f2955893bd'
imm_reader_client_id: '4fa6b522-6e6d-44c6-b023-98816b54566d'
imm_reader_client_secret: !Secret
imm_reader_subdomain: cdo-immersive-reader-<%=env%>
```

Note this would set `imm_reader_subdomain: cdo-immersive-reader-production` in `production` environment- you could override the one environment-specific variable to fix this.

Reply:
Fixed